scania-digital-design-system/sdds
# feat: introduction of conventional commits and lint-staged

**Describe pull-request**  
introduction of conventional commits and lint-staged

**Solving issue**  
This is an improvement for our commit log and faster code lint check

**How to test**  
1. Checkout branch
2. Run npm i to install new packages
3. Change some file scss or js/ts
4. Try to commit it following[ conventional-commits convention](https://www.conventionalcommits.org/en/v1.0.0/)
5. If it seems tricky, add files you want to commit and run "npm run commit" command to start commitizen questionare 
6. If all good with the commit message observe of liners will run on stages files.



**Additional context**  
Enable only the squash and merge option in GitHub. Talk about rebase.
